heroku-api-integration-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>heroku-api-parent</artifactId>
         <groupId>com.heroku.api</groupId>
-        <version>0.17-SNAPSHOT</version>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
I don't have a strong opinion either, just trying to understand. At one point we didn't want to pin the `heroku.jar` version to the API version, but don't remember why. Also, this library technically works with both v2 and v3 of the API.